0-ad-history-mod-package/simulation/templates/gaia/fruit/berry_01.xml

```diff
@@ -2,6 +2,9 @@
 <Entity parent="template_gaia_fruit">
   <Identity>
     <SpecificName>Berries</SpecificName>
+    <History>
+        Since ancient times wild berries have been a viable source of food. While caution had to be taken so that poisonous fruit were not gathered, the berries, whether they be strawberries, raspberries or blackberries were one of the few sources of sweetness. Not limited to consumption whilst fresh, berries could be dried, or squashed and their juice fermented.
```

Comment:
```suggestion
        Since ancient times, wild berries have been a viable source of food. While caution had to be taken so that poisonous fruit were not gathered, the berries, whether they be strawberries, raspberries, or blackberries, were one of the few sources of sweetness. Not limited to consumption whilst fresh, berries could be dried or squashed and their juice fermented.
```